addon/addon-test-support/@ember/test-helpers/build-owner.ts

```diff
@@ -2,14 +2,14 @@ import Application from '@ember/application';
 import type { Resolver } from '@ember/owner';
 
 import legacyBuildRegistry from './-internal/build-registry';
-import ContainerProxyMixin from '@ember/engine/-private/container-proxy-mixin';
-import RegistryProxyMixin from '@ember/engine/-private/registry-proxy-mixin';
+import ContainerProxyMixin from '@ember/-internals/runtime';
+import RegistryProxyMixin from '@ember/-internals/runtime';
 import CoreObject from '@ember/object/core';
 
 export interface Owner
   extends CoreObject,
-    ContainerProxyMixin,
-    RegistryProxyMixin {
+    ContainerProxyMixin.ContainerProxyMixin,
+    RegistryProxyMixin.RegistryProxyMixin {
```

Comment:
Then you won't need this. 😄

Reply:
LOL that makes sense. The TS error was something about namespaces which totally threw me off.